src/FixedDataTableRow.js

```diff
@@ -157,11 +170,18 @@ class FixedDataTableRowImpl extends React.Component {
         onColumnReorderEnd={this.props.onColumnReorderEnd}
         isColumnReordering={this.props.isColumnReordering}
         columnReorderingData={this.props.columnReorderingData}
-        rowHeight={this.props.height}
+        rowHeight={cellHeight}
         rowIndex={this.props.index}
       />;
     var scrollableColumnsWidth = this._getColumnsWidth(this.props.scrollableColumns);
     var columnsRightShadow = this._renderColumnsRightShadow(fixedColumnsWidth + scrollableColumnsWidth);
+    var rowExpandedStyle = {
+      top: cellHeight,
+      width: this.props.width - /* borderWidth */ 2,
```

Comment:
Just pass in the width and let the getRowExpanded method determine how they want to alter it for use case specific styling.  

Also we should move the width style out of FDT and into the example / use case specific code.  That way we don't need to worry about the border size in the library.